addons/middle-click-popup/WorkspaceQuerier.js

```diff
@@ -749,6 +751,13 @@ class TokenTypeBlock extends TokenType {
     };
 
     enumerateStringForms();
+
+    if (this.stringForms.length >= WorkspaceQuerier.MAX_STRING_FORMS) {
+      console.log(
+        "Warning: Block '" + this.block.id + "' has too many string forms. Search results may not be very good."
```

Comment:
The console wrapper works with warnings as well.
```suggestion
      console.warn(
        "Block '" + this.block.id + "' has too many string forms. Search results may not be very good."
```

Reply:
haha I actually didn't use `console.warn` for this same thing in multiple places in the original commit. I've fixed them up now. Reminds me of the classic addage;
> Ask a programmer to review 10 lines of code, they'll find 10 issues. Ask them to do 500 lines and they'll say it looks good.